semanticcpg/src/test/scala/io/shiftleft/semanticcpg/passes/CapturingLinkerTests.scala

```diff
@@ -1,29 +1,29 @@
 package io.shiftleft.semanticcpg.passes
 
-import gremlin.scala._
 import io.shiftleft.codepropertygraph.Cpg
-import io.shiftleft.codepropertygraph.generated.{EdgeTypes, NodeKeys, NodeTypes}
+import io.shiftleft.codepropertygraph.generated.{EdgeTypes, NodeKeysOdb, NodeTypes}
+import io.shiftleft.overflowdb._
 import io.shiftleft.semanticcpg.passes.linking.capturinglinker.CapturingLinker
 import io.shiftleft.semanticcpg.testfixtures.EmptyGraphFixture
 import org.scalatest.{Matchers, WordSpec}
 
+import scala.jdk.CollectionConverters._
+
 class CapturingLinkerTests extends WordSpec with Matchers {
 
   "link CLOSURE_BINDING and LOCALS with same CLOSURE_BINDING_IDs" in EmptyGraphFixture { graph =>
-    val closureBinding1 = graph + (NodeTypes.CLOSURE_BINDING, NodeKeys.CLOSURE_BINDING_ID -> "id1")
-    val closureBinding2 = graph + (NodeTypes.CLOSURE_BINDING, NodeKeys.CLOSURE_BINDING_ID -> "id2")
-    val local1 = graph + (NodeTypes.LOCAL, NodeKeys.CLOSURE_BINDING_ID -> "id1")
-    val local2 = graph + (NodeTypes.LOCAL, NodeKeys.CLOSURE_BINDING_ID -> "id2")
+    val closureBinding1 = graph + (NodeTypes.CLOSURE_BINDING, NodeKeysOdb.CLOSURE_BINDING_ID -> "id1")
```

Comment:
Are we going to keep the name `NodeKeysOdb` on the long run or is this just the case in the transition?